examples/augmented_inference.jl

```diff
@@ -0,0 +1,82 @@
+using AbstractGPs
+using TemporalGPs
+using Distributions
```

Comment:
Could you add Distributions, StatsFuns and LinearAlgebra to the Project + Manifest if they're not already there?